.github/workflows/main.yml

```diff
@@ -140,6 +141,20 @@ jobs:
         env:
           Configuration: ${{ matrix.configuration }}
 
+      - name: Determine if signing should be applaied to AppxBundle
```

Comment:
Please correct spelling of _applied_.

Reply:
It has been corrected in the very last commit 7f3048fafc0c046b65ecf813eb984efdd9d7b986.

Hmmm it should have been applied to the changes.